app/models/admin_dashboard_data.rb

```diff
@@ -207,10 +332,6 @@ def subfolder_ends_in_slash_check
     I18n.t('dashboard.subfolder_ends_in_slash') if Discourse.base_path =~ /\/$/
   end
 
-  def pop3_polling_configuration
```

Comment:
I was going to keep this validation in as a scheduled check, but there is not much point, because we already add this message to the dashboard when the pop3 poller encounters an authentication error, and that can run at a much higher frequency (default 5m)

https://github.com/discourse/discourse/blob/ad6c54839689a7f77c469abdba83e5f6a93b2af2/app/jobs/scheduled/poll_mailbox.rb#L63-L67

The only downside to this is that the message does not clear until it expires in redis. Maybe in future we should switch to doing this in a scheduled problem check instead so we can clear the error after saving the setting? Or I can do that in the same PR now :thinking: Would love an opinion.

Reply:
I think the check in poll mailbox is enough. It's ok if you change the setting and have to wait for poll mailbox to run and succeed to clear the message.